package/MDAnalysis/analysis/nuclinfo.py

```diff
@@ -763,3 +763,54 @@ def pseudo_dihe_baseflip(universe, bp1, bp2, i,
     pseudo = mdamath.dihedral(x[0] - x[1], x[1] - x[2], x[2] - x[3])
     pseudo = np.rad2deg(pseudo) % 360
     return pseudo
+
```

Comment:
Please do attend to the various PEP8 issues highlighted by pep8speaks https://github.com/MDAnalysis/mdanalysis/pull/3480#issuecomment-989321451